pkg/validate/validate.go

```diff
@@ -96,7 +96,7 @@ func Validate(ctx context.Context, spec v1alpha1.VsphereValidatorSpec, vsphereAc
 			log.Error(err, "failed to reconcile role privilege rule")
 		}
 		resp.AddResult(vrr, err)
-		log.Info("Validated privileges for account", "user", rule.Username)
+		log.Info("Validated privileges for account", "user", rule.Name())
```

Comment:
same comment about not changing this. we should be using rule.Username because its unintuitive that the rule.Name() happens to be the username